### ISSUE TITLE
README's CLI building instructions results in missing dependency failures

### ISSUE THREAD

**Issue by Bujiraso** [Astonex/Antox#689]
Hi there,
I followed the build instructions on the README.md and I am still unable to build the app, as it crashed during building due to missing dependencies (attached at the bottom).
To show that I did read the instructions, I'll write here that I did download the android SDK; I set my ANDROID_HOME and PATH, ran ``android`` and installed the latest SDK tools, build tools, and the Android Support Library; I cloned the Antox repo, downloaded the dependencies, connected my phone via USB in dev mode, and ran ./gradlew installDebug from the root of the Antox directory.

Any help would be appreciated, and if I've done everything correctly, there may be missing steps in the readme.

    > Could not resolve all dependencies for configuration ':app:_debugCompile'.
       > Could not find com.android.support:support-v4:22.1.1.
         Searched in the following locations:
             https://repo1.maven.org/maven2/com/android/support/support-v4/22.1.1/support-v4-22.1.1.pom
             https://repo1.maven.org/maven2/com/android/support/support-v4/22.1.1/support-v4-22.1.1.jar
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/support-v4/22.1.1/support-v4-22.1.1.pom
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/support-v4/22.1.1/support-v4-22.1.1.jar
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/support-v4/22.1.1/support-v4-22.1.1.pom
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/support-v4/22.1.1/support-v4-22.1.1.jar
         Required by:
             Antox:app:unspecified
             Antox:app:unspecified > com.afollestad:material-dialogs:0.7.3.0
       > Could not find com.android.support:appcompat-v7:22.1.1.
         Searched in the following locations:
             https://repo1.maven.org/maven2/com/android/support/appcompat-v7/22.1.1/appcompat-v7-22.1.1.pom
             https://repo1.maven.org/maven2/com/android/support/appcompat-v7/22.1.1/appcompat-v7-22.1.1.jar
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/appcompat-v7/22.1.1/appcompat-v7-22.1.1.pom
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/appcompat-v7/22.1.1/appcompat-v7-22.1.1.jar
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/appcompat-v7/22.1.1/appcompat-v7-22.1.1.pom
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/appcompat-v7/22.1.1/appcompat-v7-22.1.1.jar
         Required by:
             Antox:app:unspecified
             Antox:app:unspecified > com.afollestad:material-dialogs:0.7.3.0
       > Could not find com.android.support:support-v4:22.1.1.
         Searched in the following locations:
             https://repo1.maven.org/maven2/com/android/support/support-v4/22.1.1/support-v4-22.1.1.pom
             https://repo1.maven.org/maven2/com/android/support/support-v4/22.1.1/support-v4-22.1.1.jar
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/support-v4/22.1.1/support-v4-22.1.1.pom
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/support-v4/22.1.1/support-v4-22.1.1.jar
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/support-v4/22.1.1/support-v4-22.1.1.pom
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/support-v4/22.1.1/support-v4-22.1.1.jar
         Required by:
             Antox:app:unspecified > com.jpardogo.materialtabstrip:library:1.0.3
       > Could not find com.android.support:recyclerview-v7:22.1.1.
         Searched in the following locations:
             https://repo1.maven.org/maven2/com/android/support/recyclerview-v7/22.1.1/recyclerview-v7-22.1.1.pom
             https://repo1.maven.org/maven2/com/android/support/recyclerview-v7/22.1.1/recyclerview-v7-22.1.1.aar
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/recyclerview-v7/22.1.1/recyclerview-v7-22.1.1.pom
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/recyclerview-v7/22.1.1/recyclerview-v7-22.1.1.aar
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/recyclerview-v7/22.1.1/recyclerview-v7-22.1.1.pom
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/recyclerview-v7/22.1.1/recyclerview-v7-22.1.1.aar
         Required by:
             Antox:app:unspecified > com.afollestad:material-dialogs:0.7.3.0
       > Could not find com.android.support:support-annotations:22.1.1.
         Searched in the following locations:
             https://repo1.maven.org/maven2/com/android/support/support-annotations/22.1.1/support-annotations-22.1.1.pom
             https://repo1.maven.org/maven2/com/android/support/support-annotations/22.1.1/support-annotations-22.1.1.jar
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/support-annotations/22.1.1/support-annotations-22.1.1.pom
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/support-annotations/22.1.1/support-annotations-22.1.1.jar
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/support-annotations/22.1.1/support-annotations-22.1.1.pom
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/support-annotations/22.1.1/support-annotations-22.1.1.jar
         Required by:
             Antox:app:unspecified > com.afollestad:material-dialogs:0.7.3.0
       > Could not find com.android.support:support-v4:22.1.1.
         Searched in the following locations:
             https://repo1.maven.org/maven2/com/android/support/support-v4/22.1.1/support-v4-22.1.1.pom
             https://repo1.maven.org/maven2/com/android/support/support-v4/22.1.1/support-v4-22.1.1.jar
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/support-v4/22.1.1/support-v4-22.1.1.pom
             https://oss.sonatype.org/content/repositories/snapshots/com/android/support/support-v4/22.1.1/support-v4-22.1.1.jar
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/support-v4/22.1.1/support-v4-22.1.1.pom
             http://dl.bintray.com/drummer-aidan/maven/com/android/support/support-v4/22.1.1/support-v4-22.1.1.jar
         Required by:
             Antox:app:unspecified > com.github.machinarius:preferencefragment:0.1.1

**Comment by alice-margatroid** [comment]
Yes I noticed the same issue: https://github.com/Astonex/Antox/issues/693

Perhaps you can help me getting it run from https://github.com/afollestad/material-dialogs/issues/769 here?

**Comment by optimumtact** [comment]
https://github.com/subliun/Antox/issues you should file updated issues here as the development has moved to this repo
